src/presets.js

```diff
@@ -813,18 +831,22 @@ module.exports = {
 
 			async.waterfall(
 				[
-					(done) => {
+					(done) =>
+						readJSON(packageJSON, (error, pjContents) => {
+							if (error && error.code === 'ENOENT') {
+								error.message = `No such file or directory: ${packageJSON}`;
+							}
+							return done(error, pjContents);
+						}),
+					(pjContents, done) => {
+						const versionPublishedAt = new Date().toISOString();
```

Comment:
I don't mind this name at all, just wonder if `publishedAt` is enough here.

Reply:
Done. `versionist.publishedAt` is also a bit clearer than `balena.publishedAt` in relation to the fact that the "publishing" refers to a new version of the package.